index.html

```diff
@@ -19,27 +19,46 @@
     <!-- Enables media queries on mobile -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap" rel="stylesheet">
+    <style>@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap');</style>
+
     <!-- CSS-->
     <link type="text/css" rel="stylesheet" href="css/main.css" />
+    <link type="text/css" rel="stylesheet" href="css/welcome.css" />
 
     <title>Regis Altare Booster Pack Simulator</title>
   </head>
 
   <body>
     <main>
-      <h1>The Regis Altare Birthday Project 2023</h1>
 
-      <nav>
-        <a class="active" href="index.html">Collection</a>
-        <a href="pages/message-board.html">Birthday Message Board</a>
-      </nav>
+      <section id="welcome-message" class="welcome-splash visually-hidden" role="region" aria-live="polite">
+        <h2>Happy Birthday Altare!</h2>
+        <h3>Let's see what the Altventurers have made for you</h3>
+        <!-- TODO: change this to an svg or img of an arrow -->
+        <button onclick="toggleWelcomeTwo()">arrow</button>
+      </section>
 
-      <section id="card-list">
-        <!--Test image serving-->
-        <img class="card-image" src="images/elysium_delivery_services.jpg" />
-        <tcg-card></tcg-card>
+      <section id="welcome-fancy-menu" class="welcome-splash visually-hidden" role="region" aria-live="polite">
+        <img id="altare-img" src="images/AltareSplash.png" />
+        <nav id="welcome-nav">
+          <a class="special-nav-card" href="index.html">Play Gacha</a>
+          <a href="collection.html">Full Deck</a>
+          <a href="pages/messages.html">Artist/Writer Bday Messages</a>
+          <a href="pages/message-board.html">Kudoboard</a>
+          <a href="pages/credits.html">Credits</a>
+        </nav>
       </section>
 
+      <div id="loading-overlay" class="active">
+        <span>Loading</span>
```

Comment:
1. Add aria-live = polite on the loading span so that the screenreader reads it aloud, letting the user know not to interact with the page yet
2. Because the text is still in the DOM and just hidden with CSS, I can actually still "see" the "loading" text when using a screenreader
Here is me going through the main fancy menu as a screenreader user. 

https://user-images.githubusercontent.com/47371080/206876218-9bf7ba29-ac52-4ba3-b1a8-5a0d1834db60.mov

It's inaccurate that "happy birthday Altare..." and "loading" are still being read - they are not visible on the main fancy menu.

According to https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html , using `display: none` (or any of the other things he suggests) will hide it from both visually and from screenreaders.

Reply:
Ahh I see! Do you have a link to a guide on how to enable a screen reader? That way I can do some testing on my end while building also

Reply:
Mac has VoiceOver built in, which is what I was using. Windows 11 also apparently has a built in screenreader. But if you don't have those, I believe [ChromeVox](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en) is the easiest to set up, but the canonically most popular screenreaders is JAWS (have to restart your computer to get in a 30 minute timed developer mode) or NVDA.

Whichever screenreader you use has specific controls, but the main ones are the application chooser (so you can go to Chrome) and go to next/ previous item on page. Each one will also have subtle differences in reading, but we don't need to get into the weeds on that for this project imo.

Reply:
PR approval pending the `display: none` stuff o7

Reply:
I think I'm starting to understand the screen reader accessibility stuff! Being able to see it in action really helped piece things together.
I opted to use `visibility:hidden` over `display:none` since the elements that needed to be marked as hidden had transitions attached to them. I think it did the trick. Did a preliminary test using VoiceOver and it only read each section as they appeared, which is I think what we want. Also added a `visually-hidden` span before the `<nav>` to prompt the user to select a nav card. Actual content of the prompt can be changed later on of course.

Reply:
yeah! Happy to hear you got a screen reader set up. I tested it on my end as well and it sounds good. Good call on visibility instead of display.